peopleops/recruiting/HIRING_FLOW.md

```diff
@@ -0,0 +1,81 @@
+
+4) At "REVIEW PROPOSALS" step, click on "Filters". Now you should ***shortlist***(click on the thumbs-up icon) the candidates that pass the following suggested minimum criteria:
+ - Earned amount > 10k
+ - Job success > 90%
+ - Hours billed: 100+
+- English level: conversational
+- Talent type: freelancers & agencies
+
+5) Now send a message to the selected ones, asking for a quick (15 min) video chat interview. Check what timezone he's in, and use a [***time zone conversion tool***](https://savvytime.com/converter/pdt-to-ist) to make sure it will match yours.
+
+6) It's important to remember scheduling the interview time in an Alarm app (any app that remembers you about it) since once you get 2+ interviews on the same day, you'll have a hard time trying to remember when exactly did you scheduled what meeting.  Be careful to not confusing timezones!
+
+## Scheduling
+
+- Please use [Calendly](https://calendly.com/) to schedule your meetings and avoid timezone issues.
+
+
+## Interviewing
+
+- Prepare yourself to the interview. Take a look at our standard [interview questions](https://docs.google.com/document/d/130KCrw5WGYTGgJYKBVec3iEXxKkNoEkm1urHzEWznMo/edit#heading=h.7h8atrac1axk) and pick the ones that you find relevant for the situation
+
+- Communicate your supervisor about the scheduled meetings, since he should participate as well.
+
+- On this step you and your selected candidate should join a video-chat meeting room, so you can have a quick conversation and get non-verbal cues about the lead personality traits, values, etc.
+
+ - [Here's a link to countable's video-chat room](http://appear.in/countable)
+
+
+
+## Tests
+
+- Testing candidates could be done in a variety of ways and it depends upon the position you're hiring for.
+
+#### Options
+1) A small test to deliver at a pre-determined date. (For example, build the application XYZ with the following features until a certain date)
+
+2) For developers, one suggestion would be services like:
+
+    2.1) https://www.hackerrank.com
+
+
+## Contract signature
+
+- Once you and your supervisor selected a candidate, you should proceed with the contract signature. Our service of choice is [HelloSign](https://hellosign.com), that provides legally binding electronic signatures.
+
+- Ask your supervisor to review the contract with you, before sending it to your candidate.
+
+- Once everything is prepared at HelloSign, send the document to the signature to your candidate's email.
+
+- Once your contract is signed, ask your ***candidate national ID document***
+
+### Onboarding
+
+- After you've successfully completed the previous step and collected the candidate documents (save it at our [peopleops folder](https://drive.google.com/drive/folders/1HY7LPI_h5F5WxaTW5v-1_DMUXcJIjjh1) or send it to your supervisor)
+
+- You should add the candidate to the following services:
+    - Project's Trello board
+    - Slack (Ask your supervisor to add him, to avoid privacy issues)
```

Comment:
also, bitbucket, google drive folder

Reply:
Ok. What about our "initial onboarding checklist"?

Can we do something link this? https://docs.google.com/document/d/1huuidW7TyCbfkJ6HSd63s6oFF9GFdeKjFDI_1FKtXc4/edit?usp=sharing

Reply:
it's in ONBOARDING.md

Reply:
I guess we could use a separate one for short term freelancers maybe...